pkg/tsdb/azuremonitor/credentials.go

```diff
@@ -53,6 +54,8 @@ func getDefaultAzureCloud(cfg *setting.Cfg) (string, error) {
 		return azsettings.AzureUSGovernment, nil
 	case azsettings.AzureGermany:
 		return azsettings.AzureGermany, nil
+	case "AzureCustomizedCloud":
```

Comment:
you can now upgrade the sdk to v1.3.1 and use the exported variable here

Reply:
Got it, replaced all occurrences in code with the new cloud name.